sync2/tokens_table.go

```diff
@@ -258,3 +275,38 @@ func (t *TokensTable) Delete(accessTokenHash string) error {
 	}
```

Comment:
`func (t *TokensTable) Delete(accessTokenHash string) error {` is unused now then? Remove it please.